harness/determined/common/experimental/determined.py

```diff
@@ -121,6 +121,9 @@ def create_experiment(
         config: Union[str, pathlib.Path, Dict],
         model_dir: Optional[Union[str, pathlib.Path]] = None,
         includes: Optional[Iterable[Union[str, pathlib.Path]]] = None,
+        parent_id: Optional[int] = None,
```

Comment:
We expect most users to access functions in this module via the `client`: `determined.experimental.client`. This client has a (hidden) singleton `Determined` object under the hood that it passes through all its calls to.

In short, hardly anybody will ever call `Determined.create_model` directly. Usually, they'll access it via `client.create_model`.

It's more complicated than necessary (we've got a proposal somewhere to simplify it), but this currently means that every change to the overall `Determined` signature has to have a corresponding change in `determined.experimental.client`.

Reply:
Nice. Thanks. Someday I hope we don't have to do this when adding to the `Determined` class. Or at least someday have tests that check that they stay in sync.